playbooks/foreman_platform.yml

```diff
@@ -0,0 +1,13 @@
+- hosts: all
```

Comment:
But, it is "foreman_platform" as it contains Foreman + many of the most common plugins.